mmv1/third_party/terraform/services/resourcemanager/resource_google_project_service_test.go

```diff
@@ -197,6 +197,41 @@ func testAccCheckProjectService(t *testing.T, services []string, pid string, exp
 	}
 }
 
+func TestAccProjectService_checkUsageOfServices(t *testing.T) {
+	// Multiple fine-grained resources
+	acctest.SkipIfVcr(t)
+	t.Parallel()
+
+	org := envvar.GetTestOrgFromEnv(t)
+	pid := fmt.Sprintf("tf-test-%d", acctest.RandInt(t))
+	services := "bigquerystorage.googleapis.com"
+
+	acctest.VcrTest(t, resource.TestCase{
+		PreCheck:                 func() { acctest.AccTestPreCheck(t) },
+		ProtoV5ProviderFactories: acctest.ProtoV5ProviderFactories(t),
+		Steps: []resource.TestStep{
+			{
+				Config: testAccProjectService_checkUsage(services, pid, org, "false"),
+			},
+			{
+				ResourceName:            "google_project_service.test",
+				ImportState:             true,
+				ImportStateVerify:       true,
+				ImportStateVerifyIgnore: []string{"disable_on_destroy", "disable_dependent_services"},
+			},
+			{
+				Config: testAccProjectService_checkUsage(services, pid, org, "true"),
+			},
+			{
+				ResourceName:            "google_project_service.test",
+				ImportState:             true,
+				ImportStateVerify:       true,
+				ImportStateVerifyIgnore: []string{"disable_on_destroy", "disable_dependent_services"},
+			},
+		},
```

Comment:
Add another test step here that attempts to disable the service (by using an empty config) with an `ExpectError: regexp.MustCompile("[whatever is returned from the api here I'm not certain of the error message you're expecting]"),`

Reply:
> Add another test step here that attempts to disable the service (by using an empty config) with an `ExpectError: regexp.MustCompile("[whatever is returned from the api here I'm not certain of the error message you're expecting]"),`

This new flag is optional so with no input for the flag the disable will be successfull.